helix-term/src/application.rs

```diff
@@ -162,15 +162,19 @@ impl Application {
             // Unset path to prevent accidentally saving to the original tutor file.
             doc_mut!(editor).set_path(None);
         } else if !args.files.is_empty() {
-            let first = &args.files[0].0; // we know it's not empty
+            let mut files = args.files.clone();
```

Comment:
Instead of cloning here (which is a bit inefficient you can just use:


```suggestion
            let mut files = &args.files;
```

and `files = &files[1..]` instead of `remove`

Reply:
I switched to a reference to a slice. I think that gets what you are looking for with `files = &files[1..]` Then I also had to deference a variable (`pos`) in one spot further down. I'm new to Rust so please let me know if I am jumbling and mixing syntax.

Reply:
Resolving since no longer relevent.